### DISCUSSION TITLE
Using modoboa with loadbalancer

### DISCUSSION THREAD

**Discussion by AlmostArtificial** [modoboa/modoboa#2256] [General]
Hi guys,

I have a task to research the work of the next scheme:
![image](https://user-images.githubusercontent.com/34840975/120465968-38830f00-c3a7-11eb-92c7-93805ba3d3b4.png)

we need to distribute traffic within multiple local mailservers and get all the logs and graphics in one place.  
There is a situation now when our external mail provider goes offline all the mails are disappearing, so we need statistics at least.

Can the Modoboa help me with my situation? Maybe you know other solutions.

**Comment by H-A-L-I-S** [comment]
Have you tried https://github.com/modoboa/modoboa-stats already?

**Reply by AlmostArtificial** [reply]
Not yet :) now I'm just researching the information. 
Is the Modoboa suitable in my situation? I will try tomorrow.

**Reply by tonioo** [reply]
Modoboa provides graphical statistics, and also a log on incoming/outgoing emails. But, according to your drawing, it might not suit your needs since there is no builtin centralized view. I suggest you use an external tool like ELK.
